utils/ccloud_library.sh

```diff
@@ -721,6 +721,16 @@ function ccloud::create_ccloud_stack() {
   CLUSTER_NAME=demo-kafka-cluster-$SERVICE_ACCOUNT_ID
   CLUSTER_CLOUD="${CLUSTER_CLOUD:-aws}"
   CLUSTER_REGION="${CLUSTER_REGION:-us-west-2}"
+  KSQL_NAME="demo-ksql-$SERVICE_ACCOUNT_ID"
+
+  mkdir -p stack-configs
+  CLIENT_CONFIG="stack-configs/java-service-account-$SERVICE_ACCOUNT_ID.config"
+
+  ccloud::create_ccloud_stack_sub
+}
+
+function ccloud::create_ccloud_stack_sub() {
```

Comment:
Option 1: use the sub function but rename it to something like `create_ccloud_substack_per_env` ? (This may be the simplest option...just requires picking a good name)

Option 2: remove the entire idea of a sub function, and instead refactor `create_ccloud_stack` to use options, thus allowing user to pass in pre-existing service account id and environment to use.  (This is more complicated upfront but might give more freedom in the future)

cc: @rspurgeon WDYT?

Reply:
As @gianlucanatali mentioned below, we discussed out of band making the function default variables instead of breaking the function up.  He's going to work on a new verison and push a change for that.